TAScheduler/UnitTests/tests.py

```diff
@@ -10,40 +12,180 @@
 # SEE METHOD DESCRIPTIONS FOR GUIDE ON HOW TO WRITE.
 # Feel free to make suggestions on discord (add/remove/edit methods)!.
 ### Rememeber: These methods were made before any coding (I was guessing) so it's likely they should be changed.
-class TestUserLogin(TestCase):  # Alec
-    pass
-
-
-class TestUserGetID(TestCase):  # Alec
-    pass
-
-
-class TestUserGetPassword(TestCase):  # Alec
-    pass
-
-
-class TestUserGetName(TestCase):  # Alec
-    pass
-
-
-class TestUserGetRole(TestCase):  # Alec
-    pass
-
-
-class TestAdminInit(TestCase):
-    pass
-
-class TestAdminCreateCourse(TestCase):  # Alec
-    pass
-
-
-class TestAdminCreateUser(TestCase):  # Alec
-    pass
-
-
-class TestAdminCreateSection(TestCase):  # Alec
-    pass
-
+class TestUserLogin(TestCase): # Alec
+    def setUp(self):
+        user_info = User.objects.create(
+            email_address="test@example.com",
+            password="password123",
+            first_name="Test",
+            last_name="User",
+            home_address="123 Street",
+            phone_number=1234567890
+        )
+        self.userObj = UserObj(user_info)
+
+    def test_login_valid_credentials(self):
```

Comment:
UserObj is an interface. Any testing of methods that are for any methods inside the user class needed be tested from TA/Instructor/Admin